tutorials/hist/CreateROOTFile.C

```diff
@@ -0,0 +1,31 @@
+/// \file
```

Comment:
As we discussed, please remove `CreateROOTFile.C` but instead rely on `hsimple.root` which gets created during build time.

Reply:
Done